specification/recoveryservicesdatareplication/resource-manager/Microsoft.DataReplication/preview/2021-02-16-preview/recoveryservicesdatareplication.json

```diff
@@ -0,0 +1,7002 @@
+          },
+          {
+            "in": "path",
+            "name": "vaultName",
+            "description": "Vault name.",
+            "required": true,
+            "type": "string",
+            "pattern": "^[a-zA-Z0-9]*$"
+          },
+          {
+            "in": "path",
+            "name": "workflowName",
+            "description": "Workflow name.",
+            "required": true,
+            "type": "string",
+            "pattern": "^[a-zA-Z0-9]*$"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v5/types.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Returns Workflow Model on success.",
+            "schema": {
+              "$ref": "#/definitions/WorkflowModel"
+            }
+          },
+          "default": {
+            "description": "Error",
+            "schema": {
+              "$ref": "#/definitions/ArmError"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Workflow_Get": {
+            "$ref": "./examples/Workflow_Get.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.DataReplication/replicationVaults/{vaultName}/jobs": {
+      "get": {
+        "tags": [
+          "Workflow"
+        ],
+        "summary": "Lists the workflows.",
+        "description": "Gets the list of jobs in the given vault.",
+        "operationId": "Workflow_List",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v5/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v5/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "in": "path",
+            "name": "vaultName",
+            "description": "Vault name.",
+            "required": true,
+            "type": "string",
+            "pattern": "^[a-zA-Z0-9]*$"
+          },
+          {
+            "in": "query",
+            "name": "$filter",
+            "description": "Filter string.",
+            "type": "string"
+          },
+          {
+            "in": "query",
+            "name": "continuationToken",
+            "description": "Continuation token.",
+            "type": "string"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v5/types.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Returns Workflow Model Collection on success.",
+            "schema": {
+              "$ref": "#/definitions/WorkflowModelCollection"
```

Comment:
The API path is using keyword "jobs", but the data model is using "workflow", should they be unified? If so, please update other places.

Reply:
"Jobs" and "workflows" are used interchangeably. For user, it is "jobs" and for the servcie, it is "workflows". Updating all the related models from "workflow" to "job" right now is not feasible as we are on a tied schedule for public preview. We will create a work item and have the changes ready by GA as a new API release.